samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -38,6 +38,14 @@ Configuration
 
 |config|
 
+This sample configuration is split into two parts:
+* generic configuration is available in :file:`prj.conf` file
+* board specific configuration is avaialbe in :file:`boards/<BOARD>.conf` file
+
+Board specific configuration for the sample application is Bluetooth LE controller configuration.
+For :ref:`nRF5340 DK <ug_nrf5340>` Bluetooth LE controller is part of a ``child image`` aimed to run on network core.
+Configuration for a child image is stored in :file:`child_image/` subdirectory.
```

Comment:
```suggestion
Board specific configuration involves configuring the Bluetooth LE controller.
For :ref:`nRF5340 DK <ug_nrf5340>`, the Bluetooth LE controller is part of a ``child image`` aimed to run on the network core.
Configuration for the child image is stored in :file:`child_image/` subdirectory.
```

Reply:
Updated according to suggestion

Reply:
The first sentence here has one "configuration" too much, at the end of the sentence.

Reply:
I'm sorry. It should be correct now.